apps/dashboard/src/lib/components/Modal/DeleteModal.svelte

```diff
@@ -6,15 +6,16 @@
 
   export let open = false;
   export let onDelete = () => {};
+  export let isLoading = false;
 
-  function handleDelete() {
-    onDelete();
-    open = false;
+  async function handleDelete() {
+    await onDelete();
+    // Modal will be closed by the parent component after successful deletion
```

Comment:
Then this means we don't need this function handleDelete.

Why not just use the onDelete prop directly instead of creating a function in this component